### PR TITLE
Ab#6117 adjust gradients

### PR DESCRIPTION
**Pull request by matthewthomsonnz**
ADO card: ☑️ [AB#6117](https://dev.azure.com/S72/SHIFT72/_workitems/edit/6117)

- [x] Has this been discussed with Delivery Team?

Designs (note PR differs as it utilises existing image): 
- 📱 [Mobile](https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/62030a7ff375ab015a507fad)
- 💊 [Tablet](https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/62030a7c39a4d00dc252dc73)
- 🖥 [Desktop](https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/62030a7db4a419bfc4e8371e)

## Description of work

This PR:
- Implements a full width background image on meta detail page desktop
- Adjusts mobile styles to utilise the increased height between  nav and top of poster
- Image focal-point now center on mobile for consistent transition through to desktop (aligned to one side before to transition to r/h side desktop)

Above `(lg)` breakpoint (content goes from column/stacked to row/side-by-side):
- Show new desktop gradient, hide left gradient and bottom gradient
- image goes from set height, to auto height (), with 100% viewport width.
- `.meta-detail-bg` has `max-height: 900px` to ensure gradient appears behind text content)

At 0vw -> lg
- `.meta-detail-bg` and its `img` height adjusted
- purpose of this is to cater to the increased height above main content, as well as set up a better progression as the viewport width increases

Developed with design team, and they have tested and approved these changes.

## Edge cases/Caveats/Known issues
- Ultrawide/4k zooms on top center of image

## Checklist
- [x] CI tests are passing Github actions (inc. linting)
- [x] Key areas of the feature outlined for context and testing
- [x] If there are designs for this work are they noted here and in the ADO card 
- [x] Design review
- [x] Have checked this at multiple screen resolutions and range of browsers
- [ ] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [x] Updated changelog (if applicable)
- [x] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

File: site/styles/main.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
 @import '_pages';
 @import '_forms';
 @import '_card';
+@import '_gradients';
```
Comment (PeculiarGoat) [inline site/styles/main.scss:15]: Does moving this do something?  I notice that we have double ups of the classes in that file... probably a separate task to tidy that stuff up :/

File: site/styles/_variables.scss [hunk 2]
```diff
@@ -47,9 +47,9 @@
   --page-padding-top: 120px;
   --page-padding-top-md: 120px;
   --page-padding-top-lg: 180px;
-  --page-detail-padding-top: 120px;
-  --page-detail-padding-top-md: 120px;
-  --page-detail-padding-top-lg: 180px;
+  --page-detail-padding-top: 245px;
+  --page-detail-padding-top-md: 290px;
+  --page-detail-padding-top-lg: 440px;
```
Comment (PeculiarGoat) [inline site/styles/_variables.scss:52]: Are these padding values based on some sort of calculation?  Or just more random numbers?

**Review by PeculiarGoat — COMMENTED** [review]
I the the CSS looks fine.  Just got a couple of questions.

Also, do we care about the inconsistent padding between pages?  Film, TV, and bundle detail pages now have a big gap at the top, but all other pages have as small gap at the top.

File: site/styles/main.scss (hunk 1 above)
Comment (matthewthomsonnz) [reply site/styles/main.scss]: Ah no it does not, I have reverted this file to develop. I think I originally added the gradient mixin to this file and meta_item.scss came before it

File: site/styles/_variables.scss (hunk 2 above)
Comment (matthewthomsonnz) [reply site/styles/_variables.scss:52]: More random - they would be the result of matching to the designs, but then rounding to the nearest 5px

**Comment by matthewthomsonnz** [comment]
> I the the CSS looks fine. Just got a couple of questions.
> 
> Also, do we care about the inconsistent padding between pages? Film, TV, and bundle detail pages now have a big gap at the top, but all other pages have as small gap at the top.

Have asked the question to design, will post reply here

**Comment by matthewthomsonnz** [comment]
> > I the the CSS looks fine. Just got a couple of questions.
> > Also, do we care about the inconsistent padding between pages? Film, TV, and bundle detail pages now have a big gap at the top, but all other pages have as small gap at the top.
> 
> Have asked the question to design, will post reply here

@PeculiarGoat Apparently they've only been allowed to focus on key pages but its going to have a larger gap at the top. 

Theres obviously the background image on those pages but I imagine it rarely gets used? I wonder if we need to have a convo around a default generic image

**Review by PeculiarGoat — APPROVED** [review]
> only been allowed to focus on key pages
That sounds like an excuse to me lol.  We really need a design language.

Approving anyway... might as well keep this moving.

**Comment by Kazetsukai** [comment]
> > only been allowed to focus on key pages
> 
> That sounds like an excuse to me lol. We really need a design language.
> 
> Approving anyway... might as well keep this moving.

Yeah I suspect we need to revisit the frontends at some point, I think there are some other things taking priority a bit over that at the moment though

**Comment by PeculiarGoat** [comment]
Revisit with fire.

**Comment by matthewthomsonnz** [comment]
[astro.new](https://astro.new/) @Kazetsukai @PeculiarGoat

**Comment by Kazetsukai** [comment]
@matthewthomsonnz https://jdan.github.io/98.css/

**Comment by matthewthomsonnz** [comment]
> @matthewthomsonnz https://jdan.github.io/98.css/

@Kazetsukai too dated. We should go with https://botoxparty.github.io/XP.css/
